llvm/test/CodeGen/SPIRV/opencl/reflect-error.ll

```diff
@@ -1,6 +1,5 @@
-; RUN: not llc -verify-machineinstrs -O0 -mtriple=spirv64-unknown-unknown %s -o /dev/null 2>&1 | FileCheck %s
-; RUN: not llc -verify-machineinstrs -O0 -mtriple=spirv32-unknown-unknown %s -o /dev/null 2>&1 | FileCheck %s
-; UNSUPPORTED: hwasan
+; RUN: not llc -verify-machineinstrs -O0 -mtriple=spirv64-unknown-unknown %s -o - 2>&1 | FileCheck %s
```

Comment:
you don't need this patch
https://github.com/llvm/llvm-project/pull/128386/files passes: https://lab.llvm.org/buildbot/#/builders/55/builds/7459